webvtt.go

```diff
@@ -203,6 +207,48 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	return
 }
 
+// ParseTextWebVTT parses the input line to fill the Line
+func ParseTextWebVTT(i string) (o Line) {
```

Comment:
Don't export this method. 

If you're need is to be able to test it, create a `webvtt_internal_test.go' file with `package astisub` at the top. That way you'll be able to test unexported methods

Reply:
sounds good, done!